x/assetprofile/types/errors.go

```diff
@@ -9,4 +9,6 @@ var (
 	ErrAssetProfileNotFound          = errors.Register(ModuleName, 1, "asset profile not found for denom")
 	ErrChannelIdAndDenomHashMismatch = errors.Register(ModuleName, 2, "channel id and denom hash mismatch")
 	ErrNotValidIbcDenom              = errors.Register(ModuleName, 3, "not valid ibc denom")
+	ErrDecimalsInvalid               = errors.Register(ModuleName, 4, "error decimals have to be a value between 6 and 18")
+	ErrInvalidBaseDenom              = errors.Register(ModuleName, 5, "error invalid base denom")
```

Comment:
I think we can remove error prefix from all description as we are already throwing it with ErrInvalid...